src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/copy_data_entities.cljs

```diff
@@ -29,8 +29,11 @@
          (when (:server-error @state)
            [:div {:style {:marginTop "0.5em"}}
             [comps/ErrorViewer {:error (:server-error @state)}]])
-         [:div {:style {:marginTop "1em"}}
-          [comps/Button {:text "Copy"
+         [:div {:style {:display "inline-flex" :marginTop "1em"}}
+           [comps/Button {:text "Done"
+                          :style {:marginRight 20}
```

Comment:
Can we make this a more muted color so it's less easily confused with the real "I'm done with my work" button?